src/Controls/src/Core/IXmlLineInfoProvider.cs

```diff
@@ -1,3 +1,4 @@
+#nullable disable
```

Comment:
Did the .NET 6 bump have changes to nullable reference types that required this?

I don't think we did anything that sets `$(Nullable)` to `enable` by default, so wanted to check this isn't the case.

Reply:
No, this is just because I reuse this file in a net6 project which has it on.